src/contents/mod.rs

```diff
@@ -43,6 +43,12 @@ impl Content {
     }
 }
 
+impl Default for Contents {
```

Comment:
if you really need to do this - it should be reverse. Default does initialization and ::new() uses Self::default() to instantiate.